### ISSUE TITLE
React-Native: comments not supported

### ISSUE THREAD

**Issue by joeybaker** [styled-components/styled-components#917]
## Version
2.0.0

## Reproduction

```js
const A = styled.View`
  /* I'm a comment */
`

const B = () => <A />
// render this in react-native
```

## Steps to reproduce
Add any comment in a styled-component definition.

## Expected Behavior
Comments get stripped out.

## Actual Behavior
![Node of type comment not supported as an inline style](https://cloudup.com/cZpdnC4MopO+)

**Comment by k15a** [comment]
Don't know if we had a discussion about this already but it looks like a bug for me.

**Comment by jacobp100** [comment]
Iirc we used to filter it out [here](https://github.com/styled-components/styled-components/blob/master/src/models/InlineStyle.js). I'm on my phone ATM so can't check git blame

**Comment by nhducit** [comment]
still got this issue on latest version

**Comment by ckknight** [comment]
https://github.com/styled-components/styled-components/pull/1041

**Comment by mxstbr** [comment]
Fixed in #1041 by @ckknight, thanks!
